src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -206,6 +206,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
                 //     MustNotCheck,
                 // }
 
+#if PLATFORM_UNIX
```

Comment:
Right, please use partial classes or utility class that are per platform, and they would have properties like BackendSupportsCustomCertificateHandling implemented in each one.

That's the approach we're trying to use throughout. It's more maintainable than #if.